ts/parts/Utilities.ts

```diff
@@ -105,6 +105,24 @@ declare global {
         interface FormatterCallbackFunction<T> {
             (this: T): string;
         }
+        interface ModuleLoaderModuleObject {
+            resolved: boolean;
+            dependencies: Function[];
+        }
+        interface ModuleLoaderModulesLookupObject {
+            [key: string]: ModuleLoaderModuleObject;
+        }
```

Comment:
This is the same as `Dictionary<ModuleLoaderModuleObject>` if you don't need this `ModuleLoaderModulesLookupObject` to be defined in here then you could use that dictionary when needed (e.g. in `interface ModuleLoaderObject` below).